doc-theme/theme.scss

```diff
@@ -17,21 +16,32 @@ $posit-green:   #72994E;
 $posit-burgundy:#9A4665;
 $posit-burgundy-1:#78384F;
 $posit-burgundy-2:#542938;
+
+$primary: $posit-blue;
```

Comment:
@AshleyHenry15 primary is still here!

Reply:
Ah! Ignore me :)

Reply:
And, now I understand what you were saying!